marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/custom/getCustomStep.sjs

```diff
@@ -0,0 +1,21 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+var stepName;
+require('/data-hub/5/artifacts/core.sjs').getStepWithAdditionalSettings("custom", stepName)
```

Comment:
Yeah let's just implement this in here for now until it can be reused for something outside the scope of a custom step. I really want to avoid artifacts/core.sjs becoming a kitchen sink with a very broad scope.